framework/src/main/java/org/checkerframework/framework/util/AnnotatedTypes.java

```diff
@@ -963,15 +1005,13 @@ public static List<AnnotatedTypeMirror> expandVarArgsParameters(
    * @param atypeFactory the type factory to use for fetching annotated types
    * @param method the method or constructor's type
    * @param args the arguments to the method or constructor invocation
-   * @return a list of the types that the invocation arguments need to be subtype of; has the same
```

Comment:
Is this fact about length not true any more, or did you remove this because you feel the first sentence of the Javadoc explains it.

(Changes like this, and fixing spelling of Typvar to Typevar, etc., could appear in separate pull requests.)